waffle-compiler/src/generateTypes.ts

```diff
@@ -0,0 +1,15 @@
+import path from 'path';
+import {tsGenerator} from 'ts-generator';
+import {TypeChain} from 'typechain/dist/TypeChain';
+import {Config} from './config';
+
+export async function generateTypes(config: Config) {
+  const typesOutputDirName = config.typechain.outputDir || 'types';
```

Comment:
If we change it, a user has to define `outputDir`. In this case, he can just define `enable` property. It was done in order to simplicity of configuration.

Reply:
I agree that the user should just be able to set `enabled` to `true` and not care about setting the `outputDir`. However the place where we defined all default configuration values so far is the `defaultConfig.ts` file. If you apply the suggestions from https://github.com/EthWorks/Waffle/pull/383#discussion_r546392318 and https://github.com/EthWorks/Waffle/pull/383#discussion_r546392229 it will work the same way as it currently does.